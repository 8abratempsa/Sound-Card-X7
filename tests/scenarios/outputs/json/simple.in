octosql "SELECT * FROM fixtures/simple.json" --output json
