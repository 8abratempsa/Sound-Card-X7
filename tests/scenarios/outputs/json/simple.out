{"simple.field1":"value","simple.field2":42,"simple.field3":{"field4":"eulav","field5":24,"field6":null},"simple.field7":[42,43,{"field8":"value8"}]}
{"simple.field1":"value","simple.field2":42,"simple.field3":{"field4":null,"field5":"eulav","field6":"value"},"simple.field7":null}
